README.md

```diff
@@ -35,60 +35,62 @@ to use any of the available tasks and is encouraged to improve the, develop new
 
 ## Package Overview
 
-**`eo-learn`** is divided into several subpackages according to different functionalities and external package dependencies. Therefore it is not necessary for user to install entire package but only the parts that he needs.
+**`eo-learn`** package is structured into several topical subfolders according to different functionalities. Some subfolders contain extension modules under `extra` subfolder. Those modules typically require additional package dependencies which don't get installed by default, since they are usually very specific to the task.
 
-At the moment there are the following subpackages:
+The subfolders are:
 
-- **`eo-learn-core`** - The main subpackage which implements basic building blocks (`EOPatch`, `EOTask` and `EOWorkflow`) and commonly used functionalities.
-- **`eo-learn-coregistration`** - The subpackage that deals with image co-registration.
-- **`eo-learn-features`** - A collection of utilities for extracting data properties and feature manipulation.
-- **`eo-learn-geometry`** - Geometry subpackage used for geometric transformation and conversion between vector and raster data.
-- **`eo-learn-io`** - Input/output subpackage that deals with obtaining data from Sentinel Hub services or saving and loading data locally.
-- **`eo-learn-mask`** - The subpackage used for masking of data and calculation of cloud masks.
-- **`eo-learn-ml-tools`** - Various tools that can be used before or after the machine learning process.
-- **`eo-learn-visualization`** - Visualization tools for core elements of eo-learn.
+- **`core`** - The main folder which implements basic building blocks (`EOPatch`, `EOTask` and `EOWorkflow`) and commonly used functionalities.
+- **`coregistration`** - Tasks which deal with image co-registration.
+- **`features`** - A collection of utilities for extracting data properties and feature manipulation.
+- **`geometry`** - Geometry-related tasks used for transformation and conversion between vector and raster data.
+- **`io`** - Input/output tasks that deal with obtaining data from Sentinel Hub services or saving and loading data locally.
+- **`mask`** - Tasks used for masking of data and calculation of cloud/snow/other masks.
+- **`ml-tools`** - Various tools that can be used before or after the machine learning process.
+- **`visualization`** - Visualization tools for the core elements of eo-learn.
 
 ## Installation
 
-### PyPi distribution
+### Requirements
 
-The package requires Python version **>=3.8** . It can be installed with:
+Before installing `eo-learn` on **Linux** it is recommended to install the following system libraries:
```

Comment:
should these be names `OS specific requirements` or something?

Reply:
I added sections for Linux/Mac OS and Windows